pure_changelog.md

```diff
@@ -54,6 +54,7 @@
 * Fix for country code in new contact based on deleted one
 * Fixed the data displayed on snoozed alarms screen
 * Fixed displaying an incorrect window after double ending a phone call
+* Fixed time should not include 0 at the beginning
```

Comment:
This sentence has de facto negated meaning. Should be e.g.: _Fixed (removed) redundant leading zero from time representation unless exactly midnight_

(this is visible to the clients, thus I'm requesting changes)